source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_env.py

```diff
@@ -93,8 +93,6 @@ def __init__(self, cfg: ManagerBasedEnvCfg):
         print(f"\tPhysics step-size     : {self.physics_dt}")
         print(f"\tRendering step-size   : {self.physics_dt * self.cfg.sim.render_interval}")
         print(f"\tEnvironment step-size : {self.step_dt}")
```

Comment:
Might be still useful to see the device being used primarily?

```suggestion
        print(f"\tEnvironment step-size : {self.step_dt}")
        print(f"\tEnvironment device: {self.device}")
```

Reply:
The device is printed above